pkg/sensors/exec/execvemap/execve.go

```diff
@@ -15,20 +15,15 @@ type ExecveKey struct {
 	Pid uint32
 }
 
-type ExecveValueL struct {
-	Common       processapi.MsgCommon
-	Kube         processapi.MsgK8s
-	Parent       processapi.MsgExecveKey
-	ParentFlags  uint64
-	Capabilities processapi.MsgCapabilities
-}
-
```

Comment:
The removal of `ExecveValueL` is actually a code cleanup. Can you that to a separate commit in the same PR?